src/v.ts

```diff
@@ -0,0 +1,41 @@
+const completionSpec: Fig.Spec = {
+  name: "v",
+  description: "Vbuilder cli spec",
```

Comment:
This should be a description of what the cli tool does, not what spec is this 
```suggestion
  description: "Vbuilder cli spec",
```